src/main/java/org/mastodon/views/trackscheme/display/ShowSelectedTracksActions.java

```diff
@@ -192,20 +192,50 @@ private void addEdgeTargets( final RefSet< TrackSchemeVertex > selected,
 		viewGraph.releaseRef( targetRef );
 	}
 
+	/**
+	 * @return the root nodes of the selected nodes, i.e. the nodes that are not
+	 * 	   descendants of other selected nodes. The order of the nodes in the returned
+	 * 	   list is from left to right as in the TrackScheme.
+	 */
 	private RefList< TrackSchemeVertex > filterRootNodes( final RefSet< TrackSchemeVertex > selectedVertices )
 	{
+		// Note: This method finds the root nodes of the nodes in the selectedVertices set.
+		// It does so by iterating over the graph in depth-first order and from left to right.
```

Comment:
```suggestion
		// It does so by iterating over the graph in depth-first order starting from each root. The root nodes are visited in left to right order according to the TrackScheme.
```

Reply:
I changed to comment as suggested. But now the method itself is a bit different. Could check if the current comment works for the updated code.